sql-ballerina/tests/call-procedures-test.bal

```diff
@@ -0,0 +1,399 @@
+}
+
+@test:Config {
+    groups: ["procedures"],
+    dependsOn: [testCallWithStringTypesOutParams,testCreateProcedures3]
+}
+function testCallWithNumericTypesOutParams() {
+    IntegerValue paraID = new(1);
+    IntegerOutParameter paraInt = new;
+    BigIntOutParameter paraBigInt = new;
+    SmallIntOutParameter paraSmallInt = new;
+    SmallIntOutParameter paraTinyInt = new;
+    BitOutParameter paraBit = new;
+    DecimalOutParameter paraDecimal = new;
+    NumericOutParameter paraNumeric = new;
+    FloatOutParameter paraFloat = new;
+    RealOutParameter paraReal = new;
+    DoubleOutParameter paraDouble = new;
+
+    ParameterizedCallQuery callProcedureQuery = `call SelectNumericDataWithOutParams(${paraID}, ${paraInt},
+                        ${paraBigInt}, ${paraSmallInt}, ${paraTinyInt}, ${paraBit}, ${paraDecimal}, ${paraNumeric},
+                        ${paraFloat}, ${paraReal}, ${paraDouble})`;
+
+    ProcedureCallResult ret = procedureQueryMockClient(callProcedureQuery);
+    checkpanic ret.close();
+
+    decimal paraDecimalVal= 1234.56;
+
+    test:assertEquals(paraInt.get(int), 2147483647, "2nd out parameter of procedure did not match.");
+    test:assertEquals(paraBigInt.get(int), 9223372036854774807, "3rd out parameter of procedure did not match.");
+    test:assertEquals(paraSmallInt.get(int), 32767, "4th out parameter of procedure did not match.");
+    test:assertEquals(paraTinyInt.get(int), 127, "5th out parameter of procedure did not match.");
+    test:assertEquals(paraBit.get(boolean), true, "6th out parameter of procedure did not match.");
+    test:assertEquals(paraDecimal.get(decimal), paraDecimalVal, "7th out parameter of procedure did not match.");
+    test:assertEquals(paraNumeric.get(decimal), paraDecimalVal, "8th out parameter of procedure did not match.");
+    test:assertTrue((checkpanic paraFloat.get(float)) > 1234.0, "9th out parameter of procedure did not match.");
+    test:assertTrue((checkpanic paraReal.get(float)) > 1234.0, "10th out parameter of procedure did not match.");
+    test:assertEquals(paraDouble.get(float), 1234.56, "11th out parameter of procedure did not match.");
+}
+
+@test:Config {
+    groups: ["procedures"],
+    dependsOn: [testCallWithNumericTypesOutParams,testCreateProcedures4]
+}
+function testCallWithStringTypesInoutParams() {
+    IntegerValue paraID = new(1);
+    InOutParameter paraVarchar = new("test varchar");
+    InOutParameter paraCharmax = new("test char");
+    InOutParameter paraChar = new("T");
+    InOutParameter paraCharactermax = new("test c_max");
+    InOutParameter paraCharacter = new("C");
+    InOutParameter paraNvarcharmax = new("test_nchar");
+
+    ParameterizedCallQuery callProcedureQuery = `call SelectStringDataWithInoutParams(${paraID}, ${paraVarchar},
+                             ${paraCharmax}, ${paraChar}, ${paraCharactermax}, ${paraCharacter}, ${paraNvarcharmax})`;
+
+    ProcedureCallResult ret = procedureQueryMockClient(callProcedureQuery);
```

Comment:
```suggestion
    ProcedureCallResult ret = getProcedureCallResultFromMockClient(callProcedureQuery);
```

Reply:
I changed the method name.
Commit id is abf3a5ce8b2529507bfebf35c4b2d9aa392292c7